extensions/mongoProcessor/MongoQueueProcessor.js

```diff
@@ -476,10 +484,25 @@ class MongoQueueProcessor {
         });
     }
 
+    _sendMetrics() {
+        Object.keys(this._accruedMetrics).forEach(loc => {
+            const count = this._accruedMetrics[loc];
+
+            if (count > 0) {
```

Comment:
Can you add a one liner saying what it means by `count > 0`? Something like `> 0 would mean some metrics were recorded`.